src/interface/container_collection.hpp

```diff
@@ -19,16 +19,21 @@
 
 #include "interface/container.hpp"
 #include "interface/metadata.hpp"
+#include "interface/swarm_container.hpp"
 
 namespace parthenon {
 
 template <typename T>
 class ContainerCollection {
  public:
   ContainerCollection() {
-    containers_["base"] = std::make_shared<Container<T>>(); // always add "base" container
+    // Always add "base" containers
+    containers_["base"] = std::make_shared<Container<T>>();
```

Comment:
Should we rename this `VarContainer` and SwarmContainer`. It seems weird that containers of variables are special in that they don't contain swarms.

Alternatively, could the swarm live in just a normal container, but we extend the container to own swarms, in the same way as it owns `FaceVariable`s?

Reply:
I agree with you that something should be done about this. I'm not sure what the best path forward right now is (in particular I don't want to mess up how (Var)Containers already work), but both your ideas sound good to me.

Reply:
I think my preference would be to move swarms in inside container. But curious what others think. @nmsriram @jdolence @pgrete

Reply:
From discussions with @Yurlungur it sounds like the best path forward is to defer this change for later due to potential issues with mismatched integrator stages for particles and fields.